django_enum_choices/tests/test_model_integrations.py

```diff
@@ -46,11 +46,11 @@ def test_can_filter_by_enumeration_with_char_base(self):
         first_qs = StringEnumeratedModel.objects.filter(enumeration=CharTestEnum.FIRST)
         second_qs = StringEnumeratedModel.objects.filter(enumeration=CharTestEnum.SECOND)
 
-        self.assertIn(first, first_qs)
-        self.assertNotIn(second, first_qs)
+        self.assertIn(first, list(first_qs))
```

Comment:
Why do we need those lists :thinking:

Reply:
Django 1.11 throws `Object of type QuerySet is not iterable`